integration-tests/src/test/groovy/com/okta/sdk/tests/it/FeaturesIT.groovy

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.sdk.tests.it
+
+import com.okta.sdk.resource.ResourceException
+import com.okta.sdk.resource.common.EnabledStatus
+import com.okta.sdk.resource.feature.Feature
+import com.okta.sdk.resource.feature.FeatureList
+import com.okta.sdk.tests.it.util.ITSupport
+import org.testng.Assert
+import org.testng.annotations.Test
+import wiremock.org.apache.commons.lang3.RandomStringUtils
+
+import static org.hamcrest.MatcherAssert.assertThat
+import static org.hamcrest.Matchers.*
+
+/**
+ * Tests for /api/v1/features
+ * @since 2.0.0
+ */
+class FeaturesIT extends ITSupport {
+
+    @Test
+    void featureOperationsTest() {
+        // list features
+        FeatureList featureList = client.listFeatures()
+        assertThat(featureList, iterableWithSize(greaterThanOrEqualTo(1)))
```

Comment:
I'm not familiar with the features api, but this looks like a fragile test.
Mutating the state of features could change the way the org works. IIRC some of the changes are async, and some of them can only be enabled (and never disabled).  You would need to pick a specific feature to toggle.

This test also change the global state of an org, so it would not be possible to run two sets of test against the same org at once.
My suggestion would be to turn this into a UT (and not hit a live server, your call)

Reply:
That's a fair point. Will consider moving it to a UT.

Reply:
Updated with a UT instead.